docs/installation.md

```diff
@@ -11,6 +11,15 @@ brew update
 brew install arduino-cli
 ```
 
+## Install using mise
+
+If you are an enthusiast user of [mise](https://github.com/jdx/mise), the polyglot tool version manager, you can install
+the version you want by using a command like (here version is latest) :
+
+```sh
```

Comment:
````suggestion
```
````

The `sh` language identifier should only be used in code blocks that consist of shell script code. It should not be used for general command line snippets as this can result in nonsensical syntax highlighting.

I am aware that this is already done for the prior brew snippet, and it completely makes sense to follow the conventions already established in the surrounding codebase even when those conventions are less than ideal. However, the `sh` identifier is not used in the other two command line snippets in the file, so the brew snippet is the exception rather than the rule.